source/faq.txt

```diff
@@ -1,8 +1,8 @@
 .. _csharp-faq:
 
-===
-FAQ
-===
+==============
+Technical FAQ
```

Comment:
Probably simpler to just rename this 'Frequently Asked Questions'? I would guess it's implied that these are technical FAQs 

```suggestion
Frequently Asked Questions
```

Reply:
Hi @mongoKart, the purpose of lengthening the titles is to make them more descriptive and unique rather than just increasing the length ([guidance wiki here](https://wiki.corp.mongodb.com/display/DE/Content+Maintenance#:~:text=within%203%20months-,Too%20Short,-what%20they%20are). 
What would you think of FAQ - General instead? Thanks!

Reply:
My gut says that 'Frequently Asked Questions' is well understood enough--but this page will likely be eliminated during C# comprehensive coverage, so I don't think it matters much for the time being 😃